src/java/org/apache/cassandra/concurrent/SEPWorker.java

```diff
@@ -101,7 +101,7 @@ public void run()
                 if (assigned == null)
                     continue;
                 if (SET_THREAD_NAME)
-                    Thread.currentThread().setName(assigned.name + "-" + workerId);
+                    Thread.currentThread().setName(NamedThreadFactory.globalPrefix() + assigned.name + "-" + workerId);
```

Comment:
jvm-dtest thing... logs are easier to read when the thread is prefixed with the right info; this is a `""` outside jvm-dtest